glean_parser/__main__.py

```diff
@@ -74,6 +75,14 @@
     type=click.INT,
     required=False,
 )
+@click.option(
+    "--list-options",
```

Comment:
Any chance we can just make this part of the `--help` output? Having yet another option for this seems unexpected.

Reply:
for when you get back, I did some investigation and here's what I worked out:

- I _can_ hijack the help, but it's not so trivial. I was having loads of issue actually accessing the commands & even if I could I would have to programmatically build it but not so difficult. Kind of a time sink working out how to get the cli commands for the subcommand.
- In general help seems really hidden; it would be really nice if I could just chuck the extra text at the end. This is one of those cases where pip versus shipping a package is a real bummer. probably solvable with time
- I was able to enable -h for help easily
- I have removed --list-options & instead used that pattern such that if you type this command you will get the view in hopes this represents an ergonomic improvement over having two options

`glean_parser translate --options help`